components/camel-salesforce/camel-salesforce-component/src/main/java/org/apache/camel/component/salesforce/internal/streaming/SubscriptionHelper.java

```diff
@@ -16,6 +16,16 @@
  */
 package org.apache.camel.component.salesforce.internal.streaming;
 
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
```

Comment:
Please run the build with the sourcecheck profile enabled, the static part must be at the end of the imports listing.